progfun/funcional_teoria/index.html

```diff
@@ -532,6 +543,11 @@ <h3 class="text-center">Desventajas</h3>
                                 funcional no es fácil crear componentes homólogos inmutables con la misma
                                 eficiencia.
                             </li>
+                            <li>
+                                El uso de I/O (Archivos, Bases de datos y GUIs) se vuelve un problema. La manipulación de estas entradas y salidas requiere 
+                                del uso de funciones netamente impuras. Un programa que provea utilidad al usuario requerirá, en la gran mayoría de casos, 
```

Comment:
```suggestion
                                de funciones netamente impuras. Un programa que provea utilidad al usuario necesitará, en la gran mayoría de casos, 
```
Eliminación de palabras repetidas ("el uso", "requerir").